litmus-portal/backend/graphql-server/pkg/file_handlers/file_handlers.go

```diff
@@ -0,0 +1,84 @@
+package file_handlers
```

Comment:
I move this file from handler/handler.go to file_handlers/file_handlers.go because it creates some confusion between HTTP pkg which is also using the same dir structure.